server/apps/install_app.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package apps
+
+import (
+	"errors"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
+)
+
+type InInstallApp struct {
+	ActingMattermostUserID string
+	NoUserConsentForOAuth2 bool
```

Comment:
Why is this a negative boolean as opposed to `NeedsUserConsentForOAuth2` or something similar?

Reply:
default == nil value, so negate the name instead?